build.rs

```diff
@@ -66,5 +66,5 @@ fn main() {
     // Rust hides linker warnings meaning mistakes may go unnoticed.
     // Turning them into errors forces them to be displayed (and the build to fail).
     // If we do want to ignore specific warnings then `/IGNORE:` should be used.
-    println!("cargo:rustc-link-arg-bin=rustup-init=/WX");
+    println!("cargo:cargo:rustc-link-arg-bin=rustup-init=/WX");
```

Comment:
But this seems to bring back the typo?

Reply:
Nice catch! That's a Vim buffer incoherence problem on my side 🤦 Will fix.

Reply:
@djc Fixed!